README.md

```diff
@@ -89,6 +119,16 @@ Use a custom file for versioning
     version-file: 'my-custom-file.yaml'
 ```
 
+Use a pre-commit hook
+
+```yaml
+- name: Conventional Changelog Action
+  uses: TriPSs/conventional-changelog-action@v3
+  with:
+    github-token: ${{ secrets.github_token }}
+    pre-commit: ${{ github.workspace }}/some/path/pre-commit.js # requires absolute path
```

Comment:
Would `./some/path/pre-commit.js` not also work? As we also do that with the `version-file`

Reply:
Yep, done